src/command_modules/azure-cli-rdbms/setup.py

```diff
@@ -12,7 +12,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "0.2.3"
+VERSION = "0.2.4"
```

Comment:
Additional question (non-blocking). Is the service GA now? If so, when will the version be bumped to `1.x.x`

Reply:
GA modules in the CLI start at version 2.0.0.

Reply:
I will work with our PM's to check if we need to bump the version to 1.0 or 2.0 as service is GA now. This is not applicable for this PR